frontend/src/interfaces/Nft.ts

```diff
@@ -23,6 +23,26 @@ export type NftType = 'weapon' | 'character' | 'shield';
 
 export const allNftTypes: NftType[] = ['weapon', 'character', 'shield'];
 
+export interface transferedNft {
```

Comment:
A really minor thing, interfaces should be kept in Pascal case, so, TransferedNft and NftTransfer :)